packages/fiori/src/DynamicPage.hbs

```diff
@@ -1,19 +1,40 @@
 <div class="{{classes.root}}">
   <div class="{{classes.headerWrapper}}">
     <slot name="titleArea"></slot>
-    {{#unless headerSnapped}}
-      <slot name="headerArea"></slot>
-    {{/unless}}
+    {{#if showHeaderInStickArea}}
+      {{#unless headerSnapped}}
+        <slot name="headerArea"></slot>
+            <ui5-dynamic-page-header-actions
+            ?snapped="{{headerSnapped}}"
+            ?pinned="{{headerPinned}}"
+            @ui5-expand-button-click={{onExpandClick}}
+            @ui5-pin-button-click={{onPinClick}}
+          ></ui5-dynamic-page-header-actions>
```

Comment:
We can extract the following code using the inline annotation an reuse it accross the different areas:

```html
...
{{#unless headerSnapped}}
        <slot name="headerArea"></slot>
         {{> expandPinActions}}
{{/unless}}
...

{{#*inline "expandPinActions"}}
 <ui5-dynamic-page-header-actions
    ?snapped="{{headerSnapped}}"
    ?pinned="{{headerPinned}}"
    @ui5-expand-button-click={{onExpandClick}}
    @ui5-pin-button-click={{onPinClick}}
  ></ui5-dynamic-page-header-actions>
{{/inline}}
```